getContentfulEnvironment.js

```diff
@@ -6,7 +6,7 @@ const contentful = require('contentful-management');
  * (use `utils/contentful.ts` for the content delivery API client) */
 module.exports = function () {
   const contentfulClient = contentful.createClient({
-    accessToken: process.env.CONTENTFUL_MANAGEMENT_API_ACCESS_TOKEN,
+    accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
```

Comment:
Because `contentful-typescript-codegen` requires elevated API privileges, I created a separate management key for codegen. We'll want to keep the `CONTENTFUL_MANAGEMENT_API_ACCESS_TOKEN` as the `accessToken` here. If you need to do codegen let me know and I can give you a management token.

```suggestion
    accessToken: process.env.CONTENTFUL_MANAGEMENT_API_ACCESS_TOKEN,
```